frontend/src/app/home/pages/profile/profile.component.html

```diff
@@ -46,7 +46,7 @@ <h2 id="title">
                 </div>
                 <div class="info">
                     <h2 id="title"> 
-                        <mat-icon>location_on</mat-icon>
+                        <mat-icon style="margin-right:10px">location_on</mat-icon>
```

Comment:
Hardcoded? This would be a simple:
.title > .newClass {
   margin-right: 10px;
}